airgun/views/subscription.py

```diff
@@ -104,6 +104,15 @@ def is_displayed(self):
         return self.browser.wait_for_element(
             'div#subscriptions-table', timeout=10, exception=False) is not None
 
+    def search(self, query):
```

Comment:
Is this search still needed? Looks pretty much similar to default one

Reply:
search is needed, as when there is no manifest the entity is searchable but there is no table element and the base function fail in that table element not found.